packages/product-json-to-csv/src/map-product-data.js

```diff
@@ -140,7 +140,7 @@ export default class ProductMapping {
               // check if product is masterVariant
               if (product.productType)
                 product.productType.attributes.forEach((attribute: Object) => {
-                  if (!acc[attribute.name]) acc[attribute.name] = ''
+                  if (isNil(acc[attribute.name])) acc[attribute.name] = ''
```

Comment:
Nice one 👍 👏 

For future reference I would also add a comment here, in the PR, to let readers know why this change is made.

Reply:
I would additionally add why _mutation_ is necessary here.

Reply:
`if (!acc[attribute.name]) acc[attribute.name] = ''` will return empty string for `undefined`, `null` and `'0'`, that is why the '0' value is not shown in the csv product export.

Reply:
`isNil('0')` doesn't return `true` does it?

![image](https://user-images.githubusercontent.com/462507/43786994-41a02636-9a6a-11e8-82cc-b0e0ad94f224.png)

Reply:
Sorry my bad, I meant zero value (0).

Reply:
`if (isNil(acc[attribute.name])) acc[attribute.name] = ' '` was used because it is short and compact and it replaces undefined and null with empty string.

Would you prefer the below ?
`  if ( acc[attribute.name] === undefined ||  acc[attribute.name] === null )
                    acc[attribute.name] = ' ' `